recipe/meta.yaml

```diff
@@ -1,25 +1,26 @@
-{% set version = "3.4.8" %}
-{% set sha_value = "6402617c3e369a25181e1869bcf7486f861341a2531c43c4b7476fdef6b2f5d5" %}
+{% set version = "3.4.9" %}
+{% set sha_value = "b13e5c0ee5ff2945920c6b401d57121962cec92dcb6ac9bc70d6d499d8f6c256" %}
 #NB get sha256 with: curl -sL <URL> | openssl sha256
 
 package:
   name: mcstas-suite
   version: {{ version }}
 
 source:
-  url: https://github.com/McStasMcXtrace/McCode/archive/mcstas-{{ version }}.tar.gz
+  url: https://github.com/willend/McCode/archive/mcstas-{{ version }}.tar.gz
```

Comment:
det her skal selvfølgelig tilbage til den rette url.

Reply:
Yes. Ville bare lige være enig med dig om det grundlæggende før jeg lavede et nyt tag på oprindeligt repo. :-) 

(Og det er jo så der hvor vi kan vælge om vi laver nyt tag eller merger først - det ene eller det andet vil i første omgang give en "fejl" tænker jeg)

Reply:
Resolving this one very last